lib/xilinx/bitstream_reader.cc

```diff
@@ -0,0 +1,5 @@
+#include <prjxray/xilinx/bitstream_reader.h>
```

Comment:
Why does this file exist?

Reply:
Isn't needed anymore, move the content to the header after templating.